mkdocs.yml

```diff
@@ -46,13 +53,22 @@ plugins:
         - "**/.ipynb_checkpoints/*"
 
 nav:
-  - OVERVIEW:
-      - "Readme": "index.md"
-      - "Getting started": "tutorials/getting-started.ipynb"
-      - "Resources": "resources.md"
-      - "Contributing": "contributing/index.md"
+  - "What is earthaccess?": "index.md"
+  - "Quick start": "quick-start.md"
+  - "Work with us":
+      - "contributing/index.md"
+      - "Development": "contributing/development.md"
+      - "Releasing": "contributing/releasing.md"
+      - "Our meet-ups": "contributing/our-meet-ups.md"
+  - "Resources": "resources.md"
+  - USER GUIDE:
+      - "user_guide/index.md"
+      - "Authentication": "user_guide/authenticate.md"
+      - "Search": "user_guide/search.md"
+      - "Access": "user_guide/access.md"
   - HOW-TO:
       - "Authenticate with Earthdata Login": "howto/authenticate.md"
+      - "Access data": "howto/access-data.md"
```

Comment:
I suggest moving this below the search how-to's.

Reply:
[d5b59ef](https://github.com/nsidc/earthaccess/pull/498/commits/d5b59ef89a940d74711385398807d6809580feb4)